providers/assets.go

```diff
@@ -59,7 +59,7 @@ func detectAssets(runtime *Runtime, assetCandidates []*inventory.Asset, upstream
 			Upstream: upstreamConfig,
 		})
 		if err != nil {
-			return err
```

Comment:
shall we at least log the error?

Reply:
I had this in a previous version, but then it showed up twice here:
```
→ discover related assets for 1 asset(s)
x failed to connect to asset error="rpc error: code = Unknown desc = GET https://ghcr.io/v2/mondoohq/mondoo-operator/manifests/sha256-34e4bc26bf44ae1dddab394f5e9449664d6fe3a4.sig: MANIFEST_UNKNOWN: manifest unknown" asset=
x unable to connect to asset error="rpc error: code = Unknown desc = GET https://ghcr.io/v2/mondoohq/mondoo-operator/manifests/sha256-34e4bc26bf44ae1dddab394f5e9449664d6fe3a4.sig: MANIFEST_UNKNOWN: manifest unknown"
→ synchronize assets

```

So, I removed the logging.